### ISSUE TITLE
background color missing for emoji hover

### ISSUE THREAD

**Issue by germain-gg** [vector-im/element-web#25785]
### Steps to reproduce

Regressed by https://github.com/matrix-org/matrix-react-sdk/pull/11079

1. Hover an emoji in the picker

#### What did you expect?

to have a bg

#### What happened instead?

no bg

### Will you send logs?

Yes

**Comment by germain-gg** [comment]
Fixed on develop already!
